src/Batch/FileConventions/Src/AzureBatchFileConventions/packages.config

```diff
@@ -10,5 +10,5 @@
   <package id="Microsoft.WindowsAzure.ConfigurationManager" version="1.8.0.0" targetFramework="net45" />
   <package id="Newtonsoft.Json" version="6.0.8" targetFramework="net45" />
   <package id="System.Spatial" version="5.6.4" targetFramework="net45" />
-  <package id="WindowsAzure.Storage" version="6.2.0" targetFramework="net45" />
+  <package id="WindowsAzure.Storage" version="7.2.1" targetFramework="net45" />
```

Comment:
@matthchr is correct.  This will not play nicely with the Batch .NET client's use of storage.  The two need to be updated in sync.